cool-tool.rb

```diff
@@ -6,13 +6,15 @@
   "Hello World"
 end
 
+
 class CoolScore
+
+  attr_accessor :name, :age
+
   def initialize(name, age)
     @name = name
     @age = age
   end
-
 end
 
-
-#coolScore = CoolScore.new("Bob", 30)
+#coolscore = CoolScore.new("Bob", 30)
```

Comment:
We probably don't need this line here, commented code should usually be deleted.